doc/architecture.md

```diff
@@ -99,6 +99,8 @@ The Abacus Broker supports the following flows:
 
    The broker returns dashboard URL pointing to the UI and containing the service instance ID. For this to work we use a dashboard client, whose credentials are shared between broker and UI. Credentials also contains a `resource_id` and list of plans associated to it.
 
+You can also check a [demo video](https://www.youtube.com/watch?v=mJilszUPsWY&t=3s) of the above flow.
```

Comment:
Hey, we can make it like:
```
[![Demo video](https://img.youtube.com/vi/mJilszUPsWY/0.jpg)](http://www.youtube.com/watch?v=mJilszUPsWY)
```

This will remove the 3s skip forward and add a thumbnail. It will look like this:
[![Demo video](https://img.youtube.com/vi/mJilszUPsWY/0.jpg)](http://www.youtube.com/watch?v=mJilszUPsWY)